client/src/pages/login.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from "react";
+import TextField from "@material-ui/core/TextField";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+import { Paper } from "@material-ui/core";
+import Button from "@material-ui/core/Button";
+import Alert from "@material-ui/lab/Alert";
+import Joi from "joi-browser";
+
+const Login = () => {
+  const [user, setUser] = useState({ email: "", password: "" });
+  const [errors, setErrors] = useState({});
+  const { email, password } = user;
+  const schema = {
+    email: Joi.string().required().email().label("Email"),
+    password: Joi.string().required().min(6).label("Password"),
+  };
+
+  const validate = () => {
+    const options = { abortEarly: false };
+    const { error } = Joi.validate(user, schema, options);
+    if (!error) return false;
+
+    const errors = {};
+    for (let item of error.details) errors[item.path[0]] = item.message;
+    return errors;
+  };
+
+  const validateProperty = ({ id, value }) => {
+    const obj = { [id]: value };
+    const schemaObj = { [id]: schema[id] };
+    const { error } = Joi.validate(obj, schemaObj);
+    return error ? error.details[0].message : null;
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const errors = validate();
+    setErrors({ errors: errors || {} });
+    if (errors) return;
+    console.log(user);
+    setUser({ name: "", email: "", password: "" });
+  };
+  const handleChange = ({ currentTarget }) => {
+    const newErrors = { ...errors };
+    const errorMessage = validateProperty(currentTarget);
+    if (errorMessage) newErrors[currentTarget.id] = errorMessage;
+    else delete newErrors[currentTarget.id];
+    setErrors(newErrors);
+
+    setUser({ ...user, [currentTarget.id]: currentTarget.value });
+  };
+  const addFormName = (form_name) => {
```

Comment:
```suggestion
  const addFormName = (formName) => {
```

stay consistent with naming convention - so suggesting to stay consistent with camelCase here